### PR TITLE
DOC-758: 5.8.0 Release Notes and minor tweaks

### PR DESCRIPTION
**Pull request by tylerkelly13**
Related Ticket: DOC-758

Description of Changes:
* Tidy-up for 5.8 release

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: release-notes/release-notes58.md [hunk 1]
```diff
@@ -48,33 +42,36 @@ For information on editor resizing, see: [User interface options - Resize]({{sit
...
-- changelog
+### Additional enhancements
 
-## Accompanying Premium Plugin changes
+{{site.productname}} 5.8 introduces the following minor enhancements:
 
-The following premium plugin updates were released alongside {{site.productname}} 5.8.
+- Updated the `formatter.matchFormat` API to support matching formats with variables in the `classes` property.
+- Added HTML5 `audio` and `video` elements to the default alignment formats.
+- Added support for alpha list numbering to the list properties dialog.
```
Comment (lnewson) [inline release-notes/release-notes58.md:63]: Should this be here, as I thought we were keeping these out of release notes for now?

File: release-notes/release-notes58.md [hunk 2]
```diff
@@ -161,21 +158,28 @@ For information on using premium skins and icon packs, see: [Premium Skins and I
...
+- Fixed the `LineHeight` command causing the `change` event to be fired inconsistently.
 
 ## Deprecated features
 
 The following features have been deprecated with the release of {{site.productname}} 5.8:
 
-- [](#).
-
-### The...
+The undocumented `setIconStroke` Split Toolbar Button API has been deprecated and will be removed in a future release.
```
Comment (lnewson) [inline release-notes/release-notes58.md:182]:
```suggestion
- The undocumented `setIconStroke` Split Toolbar Button API has been deprecated and will be removed in a future release.
```
I believe this is meant to be a list item?

File: release-notes/release-notes58.md [hunk 3]
```diff
@@ -48,33 +42,36 @@ For information on editor resizing, see: [User interface options - Resize]({{sit
...
-## Accompanying Premium Plugin changes
+{{site.productname}} 5.8 introduces the following minor enhancements:
 
-The following premium plugin updates were released alongside {{site.productname}} 5.8.
+- Updated the `formatter.matchFormat` API to support matching formats with variables in the `classes` property.
+- Added HTML5 `audio` and `video` elements to the default alignment formats.
+- Added support for alpha list numbering to the list properties dialog.
 
-### Premium Plugin Name X.Y.Z
+## Changes in behavior
```
Comment (lnewson) [inline release-notes/release-notes58.md:65]: I don't think this is a very good/applicable heading, as changes can apply to things outside of behaviour. In this case both are changes to the UI layout which isn't a behaviour. I'm not sure what to call it other than `Changes` though.

- [ ] Come up with a better heading here please

**Review by lnewson — APPROVED** [review]

File: release-notes/release-notes58.md (hunk 2 above)
Comment (tylerkelly13) [reply release-notes/release-notes58.md:182]: One item lists are weird, so no.

File: release-notes/release-notes58.md (hunk 1 above)
Comment (tylerkelly13) [reply release-notes/release-notes58.md:63]: nope. Just not calling it out.

**Review by ashfordneil — APPROVED** [review]

File: release-notes/release-notes58.md (hunk 3 above)
Comment (tylerkelly13) [reply release-notes/release-notes58.md]: Fixed in [`ea546bd` (#1968)](https://github.com/tinymce/tinymce-docs/pull/1968/commits/ea546bd4b562cc55de94a11841bf1fd99ab727d3)

**Review by EkimChau — APPROVED** [review]

**Pull request merged**